.github/workflows/ci.yml

```diff
@@ -30,37 +35,108 @@ jobs:
           apk add \
             unzip \
       - name: Install mongodb PHP extension
-        if: matrix.php != '7.1'
+        if: (!startsWith(matrix.php, '7.1'))
         run: |
           apk add $PHPIZE_DEPS
-          pecl install mongodb-1.5.5
+          pecl install mongodb-$EXT_MONGODB_VERSION
           docker-php-ext-enable mongodb
       - name: Disable PHP memory limit
         run: echo 'memory_limit=-1' >> /usr/local/etc/php/php.ini
       - name: Install Composer
-        run: wget -qO - https://raw.githubusercontent.com/composer/getcomposer.org/fb22b78362d31c0d2bf516d1f8cdfd2745caa431/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
+        run: wget -qO - https://raw.githubusercontent.com/composer/getcomposer.org/$COMPOSER_INSTALLER_COMMIT/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
       - name: Install Symfony Flex
         run: |
           composer global require --prefer-dist --no-progress --no-suggest --ansi \
             symfony/flex
       - name: Remove Doctrine MongoDB ODM
-        if: matrix.php == '7.1'
+        if: startsWith(matrix.php, '7.1')
         run: |
           composer remove --dev --no-progress --no-update --ansi \
             doctrine/mongodb-odm \
             doctrine/mongodb-odm-bundle \
       - name: Update project dependencies
-        run: composer update --no-progress --no-suggest --ansi
+        run: |
+          mkdir -p /tmp/api-platform/core/vendor
+          ln -s /tmp/api-platform/core/vendor vendor
+          composer update --no-progress --no-suggest --ansi
       - name: Clear test app cache
-        run: tests/Fixtures/app/console cache:clear --ansi
+        run: |
+          mkdir -p /tmp/api-platform/core/var
+          ln -s /tmp/api-platform/core/var tests/Fixtures/app/var
+          tests/Fixtures/app/console cache:clear --ansi
       - name: Run PHPUnit tests
         run: vendor/bin/phpunit --colors=always
 
+  behat:
```

Comment:
remember with github actions that there is no need for all these jobs to go in one file.
you could also split them out to make a bit more readable?

`ci-behat.yml`
```
  behat:
    name: "Behat (PHP ${{ matrix.php }}) (${{ matrix.dependencies }} dependencies)"

    runs-on: ubuntu-latest

    strategy:
      matrix:
        php:
          - 7.1
          - 7.2
          - 7.3
          - 7.4.rc

        dependencies:
          - lowest
          - highest

...
```

Reply:
I disagree. It's the same CI workflow.

Reply:
yeah true, the UI would be annoying then with N different workflows.

Reply:
could still combine the behat jobs (and others) with a matrix?
or maybe the `if` exclusions would become hard to manage then

Reply:
It'll be ugly. Better to respect SRP. :laughing: (No "phpunit-or-behat" job, for example.)